EdgeModuleMessageSample/ReadMe.md

```diff
@@ -0,0 +1,51 @@
+﻿# Prerequisites for EdgeModuleMessageSample
+
+Following instructions walk through steps to set-up Azure Edge device and module.
+
+## 1. Setup and deploy custom Azure IoT Edge modules:
+
+    1. Follow the article: 
+        - https://learn.microsoft.com/azure/iot-edge/how-to-visual-studio-develop-module?view=iotedge-2020-11&pivots=iotedge-dev-cli
+
+    2. Within 'deployment.debug.template.json',  'createOptions' needs to be a string value. By default, only edgeHub module has a createOptions value which needs to be updated as a string.
+        - This createOptions is particularly important as it opens specific ports to listen to for MQTT/AMQP/HTTP connection on edge device.
+
+    3. For the new module(s) you create, update 'restartPolicy' to 'never'.
+        - Within the article example, this applies to 'IotEdgeModule1' and 'IotEdgeModule2'. 
+        - This is because you will manually connect to the module(s) from a console app in Visual Studio. If restartPolicy is set to 'always', the connection will switch between connected and disconnected as both console app and docker container will try to keep the module connected.
+
+    4. Confirm that the specific module(s) were created within the Edge device on Azure portal.
+
```

Comment:
```suggestion
1. Follow the [article](https://learn.microsoft.com/azure/iot-edge/how-to-visual-studio-develop-module?view=iotedge-2020-11&pivots=iotedge-dev-cli)

2. Within 'deployment.debug.template.json',  'createOptions' needs to be a string value. By default, only edgeHub module has a createOptions value which needs to be updated as a string.
    - This createOptions is particularly important as it opens specific ports to listen to for MQTT/AMQP/HTTP connection on edge device.

3. For the new module(s) you create, update 'restartPolicy' to 'never'.
    - Within the article example, this applies to 'IotEdgeModule1' and 'IotEdgeModule2'.  
    - This is because you will manually connect to the module(s) from a console app in Visual Studio. If restartPolicy is set to 'always', the connection will switch between connected and disconnected as both console app and docker container will try to keep the module connected.

4. Confirm that the specific module(s) were created within the Edge device on Azure portal.
```